CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 3.3.0 (2015-08-20)
+
+* (HParker / Adam Hess) add support for or-assign constants
+* (troessner) Update TooManyMethods default value to 15
```

Comment:
The default value? Not sure ... I think this falls under 'Consistency of detected smells', which only needs a point release.